### PR TITLE
Improve scancodeio installation documentation #167

### PR DESCRIPTION
**Pull request by hyounes4560**
This PR: 

- Improve the ScanCode.io installation documentation #167  

Please note that I am not able to run scripts locally to check for any errors or broken links

Signed-off-by: Hanan Younes <hyounes4560@conestogac.on.ca>

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -3,104 +3,118 @@
...
-    <https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#prerequisites>`_
+See also `ScanCode-toolkit Prerequisites <https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#prerequisites>`_ for more details.
 
-Clone the git `ScanCode.io repo <https://github.com/nexB/scancode.io>`_,
-install dependencies and create an environment file::
+* Clone the `ScanCode.io GitHub repository <https://github.com/nexB/scancode.io>`_::
 
     git clone https://github.com/nexB/scancode.io.git && cd scancode.io
+
+* Inside the ScanCode folder, install the required dependencies::
```
Comment (tdruez) [inline docs/installation.rst:55]: "Inside the ``scancode.io/`` directory, ..."

**Review by tdruez — CHANGES_REQUESTED** [review]
@hyounes4560 Thanks for those improvements to the docs, looks great.
See my minor comments for requested changes.

**Comment by tdruez** [comment]
> @hyounes4560 Please note that I am not able to run scripts locally to check for any errors or broken links

You mean you cannot compile the documentation locally on your machine?

**Comment by AyanSinhaMahapatra** [comment]
So @tdruez I think she means that she has buuld the documentation locally but can't run linter checks etc via scripts as in skeleton/other repos. https://github.com/nexB/scancode-toolkit/tree/develop/docs/scripts 

So I was wondering if we could add a github actions in scancode.io with the doc build and linter checks in a github action like we have [here](https://github.com/nexB/scancode-analyzer/blob/main/.github/workflows/ci-docs.yml) ? Would that be an okay thing to add? I guess now the doc builds are not checked for build errors or linted in scancode.io?

File: docs/installation.rst [hunk 2]
```diff
@@ -3,104 +3,118 @@
...
+    #. **Debian-based** Linux distributions
+    #. **MacOS** 10.14 and up
 
+In addition, ScanCode.io can also be run through **Docker**; this is the preferred approach **on Windows**. Refer to the :ref:`docker_image` chapter for details.
 
 .. _system_dependencies:
 
-Local installation
+.. warning::
+    ScanCode.io can **ONLY** be run on Windows through `Docker <https://www.docker.com/>`_ or `Virtual Machines <https://www.virtualbox.org/>`_. However, to avoid any installation issues, it is **NOT RECOMMEND** to run ScanCode.io on Windows machines.
```
Comment (AyanSinhaMahapatra) [inline docs/installation.rst:21]: `**NOT RECOMMEND**` should really be `*Not Recommended*` ? Avoid all caps, use the [inline markups here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup) only, consistently across documentation.

**Review by AyanSinhaMahapatra — CHANGES_REQUESTED** [review]

**Review by tdruez — APPROVED** [review]
@hyounes4560 thanks again for the documentation improvements!

@AyanSinhaMahapatra I'll add the docs linter in the CI, thanks for the input.

**Pull request merged**